meshroom/ui/reconstruction.py

```diff
@@ -1116,6 +1116,17 @@ def getPoseRT(self, viewpoint):
 
         return R, T
 
+    @Slot(QObject, result=bool)
+    def isGroupComputing(self, cameraInit):
```

Comment:
Maybe I'm missing the point but I don't really see why we need a new slot, can't we just use the `currentCameraInit.locked` property ?

Reply:
You are completely right! This has been removed and we do directly use `currentCameraInit.locked` instead.